From 4d77d8ab300f9f309a71e773ecc2064bc986d734 Mon Sep 17 00:00:00 2001
From: sprhawk <465558+sprhawk@users.noreply.github.com>
Date: Mon, 11 Feb 2019 11:45:10 +0800
Subject: [PATCH] meson generate renderer.so on macOS instead of dylib

---
 meson.build | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/meson.build b/meson.build
index c4bb1940..a29c7b95 100644
--- a/meson.build
+++ b/meson.build
@@ -1,4 +1,5 @@
 project('pitivi', 'c', version : '1.90.0.1', meson_version : '>= 0.41.0')
+host_system = host_machine.system()
 
 python = find_program('python3')
 
@@ -57,10 +58,17 @@ cdata.set('GITVERSION', GITVERSION)
 cdata.set('VERSION', meson.project_version())
 cdata.set('BUILDDIR', meson.current_build_dir())
 
+module_suffix = []
+# Python cannot import default .dylib, changed suffix to .so instead
+if ['darwin', 'ios'].contains(host_system)
+    module_suffix = 'so'
+endif
+
 coptimizations = shared_library('renderer',
     'pitivi/coptimizations/renderer.c',
     dependencies: [gst_dep, python_dep, pycairo_dep, cairo_dep],
     name_prefix : '',
+    name_suffix: module_suffix,
     install: true,
     install_dir: get_option('libdir') + '/pitivi/python/pitivi/timeline')
 
-- 
2.19.1

